ocfweb/static/js/pages/account/register.js

```diff
@@ -38,3 +39,30 @@ $(document).ready(function() {
          });
      }
 });
+
+// Recommend usernames
+function recommend() {
+    $.ajax({
+         type: 'GET',
+         url: 'recommend',
+         data: {'real_name': $('#real-name').text()},
+         success: function(data) {
+             $('#recommendations').empty();
+             var recommendations = data['recommendations'];
+             for (var i in recommendations) {
+                 var recommendation = recommendations[i];
+                 $('#recommendations').append(
+                     $('<button>', {
+                         type: 'button',
+                         class: 'list-group-item list-group-item-action list-group-item-success recommendation',
+                         onclick: '$("#id_ocf_login_name").val("' + recommendation + '").trigger("keyup");',
+                         text: recommendation,
+                     })
+                 );
+             }
+         }
+     });
+}
+$(document).ready(function() {
+    recommend();
+});
```

Comment:
Instead of having two `$(document).ready` blocks, I'd combine the two, so that this function call is in the block above. Otherwise, I think it could get confusing if someone is reading this file, since they won't expect there to be two different places that functions are called from.